### PR TITLE
Add file dropdown feature

### LINKED ISSUE #2437: Allow to drop import files on project creation
When setting an AOI for a new project, allow the user to upload a file by just dragging and dropping it over the map.

### PR DESCRIPTION
**Pull request by sumukhah**
Screenshot from 2020-10-22 19-24-42

Resolves #2437

### CODE REVIEW

**Comment by willemarcel** [comment]
@sumukhah that is nice! Do you think we could allow the user to drag the file in any part of the page or over the map?

File: frontend/src/components/projectCreate/messages.js [hunk 1]
```diff
@@ -180,6 +180,10 @@ export default defineMessages({
     id: 'management.projects.create.upload.description',
     defaultMessage: 'Import a GeoJSON, KML, OSM or zipped SHP file.',
   },
+  dragFilesDescription: {
+    id: 'management.projects.create.drag.description',
+    defaultMessage: 'Drag Files or Click to Browse',
```
Comment (willemarcel) [inline frontend/src/components/projectCreate/messages.js:185]: Could you capitalize only the D on "Drag"?

**Comment by sumukhah** [comment]
> @sumukhah that is nice! Do you think we could allow the user to drag the file in any part of the page or over the map?

No. At present user has to drag into the rectangular input area.

**Comment by sumukhah** [comment]
Created a separate PR for implementing drag and drop feature on the map (https://github.com/hotosm/tasking-manager/pull/3769)

**Review by willemarcel — APPROVED** [review]

**Comment by willemarcel** [comment]
I merged with some improvements on #3768

**Pull request closed without merging**